core/store/tx_manager.go

```diff
@@ -619,6 +619,14 @@ func (txm *EthTxManager) processAttempt(
 			"jobRunId", jobRunID,
 		)
 
+		// Update metric on here due to possible significant delay
+		// before being Safe.  There may be a large backlog of
+		// transaction confirmations.
```

Comment:
This comment doesn't make a whole lot of sense to me. Any way it can be rewritten to be clearer?

Reply:
updated, @jleeh this look like what you explained?